src/commands/groups/util.ts

```diff
@@ -0,0 +1,16 @@
+export const outputFormat = {
+	table: [
+		{
+			header: 'ID',
+			property: 'id'
+		},
+		{
+			header: 'Name',
+			property: 'name'
+		},
+		{
+			header: 'Type',
+			property: 'type'
+		}
+	]
```

Comment:
I would include `parent` in the json output format

Reply:
will do